librocksdb_sys/crocksdb/c.cc

```diff
@@ -2323,7 +2337,23 @@ void crocksdb_options_set_env(crocksdb_options_t* opt, crocksdb_env_t* env) {
   opt->rep.env = (env ? env->rep : nullptr);
 }
 
-void crocksdb_options_set_info_log(crocksdb_options_t* opt, crocksdb_logger_t* l) {
+crocksdb_logger_t* crocksdb_logger_create_from_impl(crocksdb_logger_impl_t* li) {
+  crocksdb_logger_t * logger = new crocksdb_logger_t;
+  logger->rep = std::shared_ptr<Logger>(li);
+  return logger;
+}
+
+crocksdb_logger_impl_t* crocksdb_logger_impl_create(void* rep, void (*destructor_)(void*),logv_cb logv) {
```

Comment:
Please add space between `,` and `logv_cb`.

Reply:
Done